hack/ci/run-argocd-e2e-tests.sh

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env bash
+
+# Copyright 2025 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### This script sets up a local KKP installation in kind, deploys a
+### couple of test Presets and Users and then runs the IPAM e2e tests.
+
+set -euo pipefail
+#set -x
+cd $(dirname $0)/../..
+source hack/lib.sh
+
+# if [ -z "${E2E_SSH_PUBKEY:-}" ]; then
+#   echodate "Getting default SSH pubkey for machines from Vault"
+#   retry 5 vault_ci_login
+#   E2E_SSH_PUBKEY="$(mktemp)"
+#   vault kv get -field=pubkey dev/e2e-machine-controller-ssh-key > "${E2E_SSH_PUBKEY}"
+# else
+#   E2E_SSH_PUBKEY_CONTENT="${E2E_SSH_PUBKEY}"
+#   E2E_SSH_PUBKEY="$(mktemp)"
+#   echodate "${E2E_SSH_PUBKEY_CONTENT}" > "${E2E_SSH_PUBKEY}"
+# fi
+
+# echodate "SSH public key will be $(head -c 25 ${E2E_SSH_PUBKEY})...$(tail -c 25 ${E2E_SSH_PUBKEY})"
+
+EXTRA_ARGS=""
+provider="${PROVIDER:-aws}"
```

Comment:
For clarity IMHO it makes more sense to keep this as 1 variable instead of introducing a local, lowercase version.

Reply:
Updated.. but this was also picked up from some other existing CI job. The inconsistency remains in that shell script.